### DISCUSSION TITLE
What's the difference between discussions here and other support channels?

### DISCUSSION THREAD

**Discussion by lwcorp** [sandboxie-plus/Sandboxie#1768] [General]
https://forum.xanasoft.com/#sandboxie.4 also has discussions about Sandboxie, so what is the difference? This is just to know better where to post.

**Comment by isaak654** [comment]
The main difference is that some people are not inclined to GitHub, so more support channels are available:

- A contact page is available at [Contact | Xanasoft](https://xanasoft.com/contact/)

- Steps to report a security vulnerability can be found in the [SECURITY.md](https://github.com/sandboxie-plus/Sandboxie/security/policy) file

- If you're interested to chat with the lead developer (as a contributor), there is a primary [Sandboxie-Plus Slack channel](https://github.com/sandboxie-plus/Sandboxie/issues/479) and a secondary [Discord server for Sandboxie-Plus](https://www.wilderssecurity.com/threads/discord-server-for-sandboxie-plus.445578/)

- A Sandboxie board is available on Wilders forum: https://www.wilderssecurity.com/forums/sandboxie-sbie-open-source-plus-classic.144

- A Sandboxie board is available on MalwareTips forum: https://malwaretips.com/forums/sandboxie

- The [Xanasoft forum](https://forum.xanasoft.com/) contains message support boards for all programs (including Sandboxie) developed by the same lead developer
